pkg/importer/http-datasource.go

```diff
@@ -272,6 +272,12 @@ func createHTTPReader(ctx context.Context, ep *url.URL, accessKey, secKey, certD
 		return nil, uint64(0), true, errors.Errorf("expected status code 200, got %d. Status: %s", resp.StatusCode, resp.Status)
 	}
 
+	acceptRanges, ok := resp.Header["Accept-Ranges"]
+	if !ok || acceptRanges[0] != "bytes" {
```

Comment:
Probably not a big deal, but I wonder if this condition should instead be `!= "none"`

https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests